### DISCUSSION TITLE
Student developer pack not received

### DISCUSSION THREAD

**Discussion by dev-shivam-verma** [community/community#140349] [GitHub Education]
It's the 4th day since I verified myself on GitHub education, but I haven't received any student developer pack yet!!!!!!

**Comment by Thevandieg** [comment]
Hello Shivam,

You should wait until you receive an email confirming that your student status has been verified. Once that email arrives, all the features of the GitHub Student Developer Pack will automatically be applied to your account.

A quick note of caution: when activating certain platform benefits, such as Microsoft Azure, make sure to choose the right account for the subscription. In my case, I used my school email for the Azure student subscription, but the school later deleted the account, causing me to lose access to that subscription. Fortunately, Azure has a generous free tier, so I was able to continue using it without issues.

**Comment by davevad93** [comment]
Hi @dev-shivam-verma ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @dev-shivam-verma was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
